python/lsst/ip/isr/ampOffset.py

```diff
@@ -235,16 +235,21 @@ def run(self, exposure):
             # ensuring that no values are erroneously added/subtracted.
             pedestals = np.nan_to_num(np.linalg.lstsq(A, B, rcond=None)[0])
 
-        metadata = exposure.getMetadata()
+        metadata = exposure.getMetadata()  # Exposure metadata.
+        self.metadata["AMPOFFSET_PEDESTALS"] = {}  # Task metadata.
         for amp, pedestal in zip(amps, pedestals):
             ampIm = exposure.image[amp.getBBox()].array
             ampIm -= pedestal
             ampName = amp.getName()
+            # Add the amp pedestal to the exposure metadata.
```

Comment:
Note that I didn't remove the amp offset pedestals from the exposure metadata after adding them to the task metadata.